hledger-defuns.el

```diff
@@ -177,8 +176,8 @@ not balance at point."
                (neg-amounts (seq-filter (lambda (n)
                                           (not (< 0 n)))
                                         amounts))
-               (pos-amounts-sum (reduce '+ pos-amounts :initial-value 0.0))
-               (neg-amounts-sum (reduce '+ neg-amounts :initial-value 0.0))
+               (pos-amounts-sum (cl-reduce '+ pos-amounts :initial-value 0.0))
```

Comment:
@narendraj9 

Look at the bottom of the `cl-lib.el`, you will find the expression

```elisp
(unless (load "cl-loaddefs" 'noerror 'quiet)
  ;; When bootstrapping, cl-loaddefs hasn't been built yet!
  (require 'cl-macs)
  (require 'cl-seq))
```

Basically it is trying to load `cl-loaddefs` (which in turn loads `cl-seq`) if that fails it loads `cl-seq` directly